extensions/amp-story/1.0/amp-story-open-page-attachment.js

```diff
@@ -72,7 +89,14 @@ export const buildOpenInlineAttachmentElement = (element) =>
 export const renderPageAttachmentUI = (win, pageEl, attachmentEl) => {
   const openImgAttr = attachmentEl.getAttribute('cta-image');
   const attachmentHref = attachmentEl.getAttribute('href');
-  if (isPageAttachmentUiV2ExperimentOn(win) && !attachmentHref && openImgAttr) {
+  if (isPageAttachmentUiV2ExperimentOn(win) && attachmentHref) {
+    return renderOutlinkPageAttachmentUI(
+      win,
+      pageEl,
+      attachmentEl,
+      attachmentHref
+    );
+  } else if (isPageAttachmentUiV2ExperimentOn(win) && openImgAttr) {
```

Comment:
Super nitty: 99% of the time you don't need to use any `else` statement. Because you have a return in the previous `if`, you don't need the `else` here and below
They make reading the code a bit harder because you have to look at both the previous if and the current if to know how it can get into this block